src/libs/nav.ts

```diff
@@ -73,13 +73,20 @@ export const getPath = (name: string, params?: Record<string, any>): string => {
 /**
  * alias for getPath()
  */
-export const getLink = (name: string, params?: Record<string, any>) => `#${getPath(name, params).slice(1)}`; // slice off leading slash
+export const getLink = (name: string, pathParams?: Record<string, any>, queryParams?: Record<string, any>) => {
```

Comment:
It would be nice to add return types to the signatures.

```suggestion
export const getLink = (name: string, pathParams?: Record<string, any>, queryParams?: Record<string, any>): string => {
```